geth/api/api.go

```diff
@@ -39,10 +40,15 @@ func (api *StatusAPI) NodeManager() common.NodeManager {
 }
 
 // AccountManager returns reference to account manager
-func (api *StatusAPI) AccountManager() common.AccountManager {
+func (api *StatusAPI) AccountManager() account.Accounter {
```

Comment:
Maybe it makes sense to rename it `Accounter()` too. So that externally we always talk about an `Accounter` even if the standard implementation is `account.Manager`. So even manual stubbing and injections for tests should become more simple (have to check concrete here).

Reply:
I had doubts about changing any public method on the api as they are used as bindings from java. 
If that's the case, may be better to just create a specific issue to address this on both status-react and status-go packages.

Reply:
Oh, sure, did not checked it. And yes, in case of too much dependencies this has to be changed carefully in extra smaller PRs. Good hint.